pkg/rpcclient/nns/contract.go

```diff
@@ -31,6 +36,28 @@ type ContractReader struct {
 	hash    util.Uint160
 }
 
+// Actor is used by Contract to call state-changing methods.
+type Actor interface {
+	Invoker
+
+	nep11.Actor
+
+	MakeCall(contract util.Uint160, method string, params ...any) (*transaction.Transaction, error)
+	MakeRun(script []byte) (*transaction.Transaction, error)
+	MakeUnsignedCall(contract util.Uint160, method string, attrs []transaction.Attribute, params ...any) (*transaction.Transaction, error)
+	MakeUnsignedRun(script []byte, attrs []transaction.Attribute) (*transaction.Transaction, error)
+	SendCall(contract util.Uint160, method string, params ...any) (util.Uint256, uint32, error)
+	SendRun(script []byte) (util.Uint256, uint32, error)
+}
+
+// Contract implements all contract methods.
+type Contract struct {
```

Comment:
Need to adjust documentation to `nns` package.

Reply:
Not resolved. Search for `Package nns provide some RPC wrappers for the non-native NNS contract.` and adjust this documentation.